src/stavetie.ts

```diff
@@ -35,18 +50,19 @@ export class StaveTie extends Element {
       last_x_shift: 0,
       y_shift: 7,
       tie_spacing: 0,
-      font: { family: 'Arial', size: 10, style: '' },
+      font: { family: 'Arial', size: 10, weight: '' },
     };
 
     this.font = this.render_options.font;
     this.setNotes(notes);
   }
 
-  setFont(font) {
+  setFont(font: FontInfo): this {
```

Comment:
It's good that you changed `style` to `weight` but this breaks the public interface, so add a console warning, and also set `this.font.weight = font.style`.

Reply:
I believe that I did not break the interface, I fixed a bug. The value was simply ignored. The images resulting are now different. I would not add console log, to say there was a bug.

Reply:
Oh, sorry. I thought `style` was being used. Thanks.